File: Find_the_Town_Judge/Find_the_Town_Judge.go
package main

import "fmt"

func main() {
	n := 3
	trust := [][]int{{1, 3}, {2, 3}}
	fmt.Println(findJudge(n, trust))
}

func findJudge(n int, trust [][]int) int {
	ans := make([]int,n+1)
	for _,x := range trust{
		ans[x[0]] -=1
		ans[x[1]] +=1
	}

	for i := 1; i <= n;i++{
		if (ans[i] == n-1){
			return i
		}
	}
	return -1
}

File: Find_the_Town_Judge/Find_the_Town_Judge.py
from collections import defaultdict
def findJudge(n,trust):
    res = ([0] * (n+1))
    print(res)
    for i in trust:
        res[i[0]]-=1
        res[i[1]]+=1
    print(res)
    
    for i in range(1,n+1):
        x = res[i]
        print(x)
        if (x == n-1):
            return i

    return -1



# n = 2
# trust = [[1,2]]
n = 3 
trust = [[1,3],[2,3]]
print(findJudge(n,trust))